app/controllers/survey_controller.rb

```diff
@@ -1,4 +1,10 @@
 class SurveyController < ApplicationController
   def index
+    @questionList = Question.order(:order)
+    #@questions = [{:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}, {:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}]
+    @questionList.all.each do |theQuestion|
+      theQuestion.answers = Answer.where(question_id: theQuestion.id)
```

Comment:
Do you have to do this? I'm pretty sure q.answers should already be populated if it's in the seed data file.